dotnet-desktop-guide/net/wpf/get-started/create-app-visual-studio.md

```diff
@@ -0,0 +1,247 @@
+
+- [Visual Studio 2019 version 16.9 or later versions](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019+desktopguide+wpf)
+  - Select the [Visual Studio Desktop workload](/visualstudio/install/modify-visual-studio?view=vs-2019&preserve-view=true#modify-workloads)
+  - Select the [.NET 5 individual component](/visualstudio/install/modify-visual-studio?view=vs-2019&preserve-view=true#modify-individual-components)
+
+## Create a WPF app
+
+The first step to creating a new app is opening Visual Studio and generating the app from a template.
+
+01. Open Visual Studio.
+01. Select **Create a new project**.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-create-new-project.png" alt-text="Create a new WPF project in Visual Studio 2019 for .NET":::
+
+01. In the **Search for templates** box, type **wpf**, and then press <kbd>Enter</kbd>.
+01. In the **code language** dropdown, choose **C#** or **Visual Basic**.
+01. In the templates list, select **WPF Application** and then select **Next**.
+
+    > [!IMPORTANT]
+    > Don't select the **WPF Application (.NET _Framework_)** template.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-template-search.png" alt-text="Search for the WPF template in Visual Studio 2019 for .NET":::
+
+01. In the **Configure your new project** window, do the following:
+
+    01. In the **Project name** box, enter **Names**.
+    01. Select the **Place solution and project in the same directory** check box.
+    01. Optionally, choose a different **Location** to save your code.
+    01. Select the **Next** button.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-config-new-project.png" alt-text="Configure new WPF project in Visual Studio 2019 for .NET":::
+
+01. In the **Additional information** window, select **.NET 5.0 (Current)** for **Target Framework** and make sure that the **Run on .NET Framework** check box is cleared. Select the **Create** button.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-config-new-project-next.png" alt-text="Select target framework for new WPF project in Visual Studio 2019 for .NET":::
+
+Once the app is generated, Visual Studio should open the XAML designer pane for the default window, _MainWindow_. If the designer isn't visible, double-click on the _MainWindow.xaml_ file in the **Solution Explorer** pane to open the designer.
+
+### Important parts of Visual Studio
+
+Support for WPF in Visual Studio has five important components that you'll interact with as you create an app:
+
+:::image type="content" source="media/create-app-visual-studio/vs-main-window.png" alt-text="The important components of Visual Studio you should know when creating a WPF project for .NET":::
+
+01. Solution Explorer
+
+    All of your project files, code, windows, resources, will appear in this pane.
+
+02. Properties
```

Comment:
Is this numbering correct?

Reply:
Oh wait, I'm a dork. It's in the build report. [Staging](https://review.docs.microsoft.com/en-us/dotnet/desktop/wpf/get-started/create-app-visual-studio?view=netdesktop-5.0&branch=pr-en-us-193)